test-support/helpers/ember-power-select.js

```diff
@@ -67,7 +67,7 @@ export default function() {
   const isEmberOne = Ember.VERSION.match(/1\.13/);
 
   Ember.Test.registerAsyncHelper('selectChoose', function(app, cssPath, value) {
-    const id = find(cssPath).find('.ember-power-select-trigger').attr('id').match(/ember-power-select-trigger-ember(\d+)/)[1];
+    const [,id] = find(cssPath).find('.ember-power-select-trigger').attr('id').match(/ember-power-select-trigger-ember(\d+)/);
```

Comment:
I suspect that's the case....

On Wed, Apr 27, 2016 at 2:28 PM, Miguel Camba <notifications@github.com>
wrote:

> In test-support/helpers/ember-power-select.js
> <https://github.com/cibernox/ember-power-select/pull/472#discussion_r61339710>
> :
>
> > @@ -67,7 +67,7 @@ export default function() {
> >    const isEmberOne = Ember.VERSION.match(/1\.13/);
> >
> >    Ember.Test.registerAsyncHelper('selectChoose', function(app, cssPath, value) {
> > -    const id = find(cssPath).find('.ember-power-select-trigger').attr('id').match(/ember-power-select-trigger-ember(\d+)/)[1];
> > +    const [,id] = find(cssPath).find('.ember-power-select-trigger').attr('id').match(/ember-power-select-trigger-ember(\d+)/);
>
> Maybe this kind of destructuring is the one that compiles to someting that
> uses Symbol.iterator?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/cibernox/ember-power-select/pull/472/files/45ce0beba9b5aaa513c537921d6272aa8cc2d95c#r61339710>
>